packages/main/src/TimePickerBase.ts

```diff
@@ -75,16 +75,38 @@ import ResponsivePopoverCommonCss from "./generated/themes/ResponsivePopoverComm
 *
 * @event sap.ui.webc.main.TimePickerBase#change
  * @public
- */
-@event("change")
+ * @param {string} value The submitted value.
+ * @param {boolean} valid Indicator if the value is in correct format pattern and in valid range.
+*/
+@event("change", {
+	detail: {
+		value: {
```

Comment:
If `value` event detail hasn't been documented so far, maybe we can skip it all together as we have a `value`. Until now, for all components, if the "value" can be accessed via the component instance and the property - we don't add that to the event as well.  For example, the Input component's `change` and `input`.
Of course, if that `value` in the event detail is equal to event.target.value.

Reply:
Yes at the firing point both event.detail.value and event.target.value will be equal, so I will remove the documentation part.